resources/production/chainspec.toml

```diff
@@ -148,7 +148,7 @@ call_contract = { cost = 4_500, arguments = [0, 0, 0, 0, 0, 420, 0] }
 call_versioned_contract = { cost = 200, arguments = [0, 0, 0, 0, 0, 0, 0, 0, 0] }
 create_contract_package_at_hash = { cost = 200, arguments = [0, 0] }
 create_contract_user_group = { cost = 200, arguments = [0, 0, 0, 0, 0, 0, 0, 0] }
-create_purse = { cost = 2_500_000_000, arguments = [0, 0] }
+create_purse = { cost = 170_000, arguments = [0, 0] }
```

Comment:
I think this change should be reverted as it was changed to `2_500_000_000` in #1326 recently.

Same point for `[system_costs.mint_costs.mint]` and `[system_costs.mint_costs.create]` below.

Changes to `[system_costs.auction_costs.add_bid]` and `[system_costs.auction_costs.delegate]` appear to be correct here though, having been changed to these values in #1834.

Reply:
Corrected.